psamm/commands/gapcheck.py

```diff
@@ -0,0 +1,74 @@
+# This file is part of PSAMM.
+#
+# PSAMM is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PSAMM is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PSAMM.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Copyright 2014-2016  Jon Lund Steffensen <jon_steffensen@uri.edu>
+
+from __future__ import unicode_literals
+
+import logging
+
+from ..command import Command, MetabolicMixin, SolverCommandMixin
+from ..gapfill import gapfind, GapFillError
+
+logger = logging.getLogger(__name__)
+
+
+class GapCheckCommand(MetabolicMixin, SolverCommandMixin, Command):
+    """Check for compound production gaps in model."""
+    @classmethod
+    def init_parser(cls, parser):
+        parser.add_argument(
+            '--threshold', type=float, default=1e-5,
+            help='Threshold for compound production')
```

Comment:
Do we want to start using this threshold option instead of what we were calling epsilon?
It seems to still be treated the same way as epsilon was.